airbyte-integrations/connectors/source-rki-covid/source_rki_covid/spec.json

```diff
@@ -0,0 +1,36 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Rki Covid Spec",
```

Comment:
Today the correct name of this connector is `Rki Covid Germany`. Why not create a parameter to select the country Germany and US, if in the future there is a new one would be easy to implement too.

Reply:
Well the reason is RKI means [Robert Koch-Institut COVID-19 API - von Marlon Lückert](https://api.corona-zahlen.org/docs/). Its an institute of Germany, that only deals with data for Germany. So adding a parameter is not a problem.
In future if they choose to give data for other countries then it would be easily implemented